src/__tests__/fake-timers.js

```diff
@@ -0,0 +1,85 @@
+import {render} from './helpers/test-utils'
```

Comment:
We've moved all the fake timers tests to this file. The only reason we're using fake timers is for testing timeouts and we don't want this test to take forever.